src/test/fuzz/addrdb.cpp

```diff
@@ -17,6 +17,8 @@ void test_one_input(const std::vector<uint8_t>& buffer)
 {
     FuzzedDataProvider fuzzed_data_provider(buffer.data(), buffer.size());
 
+    // `ban_entry` is no longer used, but the harness still needs to consume the
+    // same data that it did previously to not invalidate existing seeds.
```

Comment:
Maybe comment here "The point of the code is to exercise all CBanEntry ctors :)" from https://github.com/bitcoin/bitcoin/issues/19549#issuecomment-660552734